proto/viam/app/v1/billing.proto

```diff
@@ -108,6 +108,12 @@ message GetCurrentMonthUsageResponse {
   double data_egres_usage_cost = 5;
   double remote_control_usage_cost = 6;
   double standard_compute_usage_cost = 7;
+  double binary_cloud_storage_usage = 11;
```

Comment:
the numbers are weird but I wanted to keep the usage order sequential for clarity